src/portfolio.js

```diff
@@ -115,6 +115,36 @@ const techStack = {
   ]
 };
 
+
+// Your top 3 work experiences
+
+const workExperiences = {
+  viewExperiences: true, //Set it to true to show workExperiences Section
+  experience: [
+    {
+      role: "Software Engineer",  
+      company: "Facebook",
+      companylogo:  require("./assests/images/facebookLogo.png"),
+      date: "June 2018 – Present",
+      descrip: "Contribute to and build new features for the API Management Platform that include test automation frameworks to deliver high quality, maintainable code."
+    },
+    {
+      role: "Front-End Developer",   
+      company: "Quora",
+      companylogo: "https://www.referralcandy.com/wp-content/uploads/2017/06/quora-featured-image-2-740x740.png",
+      date: "May 2017 – May 2018",
+      descrip: "Write clean, maintainable code using best industry standards in Web languages, and integrate with existing APIs."
+    },
+    {
+      role: "Software Engineer Intern",  
+      company: "Airbnb",
+      companylogo: require("./assests/images/airbnbLogo.jpg"),
+      date: "Jan 2015 – Sep 2015",
+      descrip: "Develop and maintain new/current software applications, meeting user requirements and complying with technical specifications."
```

Comment:
Let's change `descrip` prop name to `desc` for consistency and readability. Also update accordingly in other files.
Switch `desc` to Lorem Ipsum

Reply:
do you want to change the date to Lorem Ipum as well?

Reply:
Nah, just add some place holder date or let it be as it is right now.